### PR TITLE
Emily dev p1 s2

### PR DESCRIPTION
**Pull request by emilybpadgett**
I completed step 2 of project 1. I added password to Datastore and updated the test for Datastore to test the password.

### CODE REVIEW

**Review by alondarobichaud — APPROVED** [review]

**Comment by tsums** [comment]
How does this PR related to #5 ? Is this an extension of it? It looks like it also contains some of the same changes.

**Comment by emilybpadgett** [comment]
This was step 2 of the project. I think I somehow messed up the branches when submitting a pull request for this. The step 2 parts are editing the PersistentDataStore class to add the password to 2 places. Then I also added tests in persistentDataStoreTest.

File: src/main/java/codeu/model/store/persistence/PersistentDataStore.java [hunk 1]
```diff
@@ -64,9 +64,12 @@ public PersistentDataStore() {
       try {
         UUID uuid = UUID.fromString((String) entity.getProperty("uuid"));
         String userName = (String) entity.getProperty("username");
+	String password = (String) entity.getProperty("password");
```
Comment (tsums) [inline src/main/java/codeu/model/store/persistence/PersistentDataStore.java:67]: Nit: Fix this indentation.

File: src/main/java/codeu/model/store/persistence/PersistentDataStore.java [hunk 2]
```diff
@@ -64,9 +64,12 @@ public PersistentDataStore() {
...
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
-        //TODO: get password from Datastore
-        User user = new User(uuid, userName, null, creationTime);
+
```
Comment (tsums) [inline src/main/java/codeu/model/store/persistence/PersistentDataStore.java]: Nit: You can remove this extra blank line here.

**Review by tsums — CHANGES_REQUESTED** [review]

File: src/main/java/codeu/model/store/persistence/PersistentDataStore.java (hunk 1 above)
Comment (emilybpadgett) [reply src/main/java/codeu/model/store/persistence/PersistentDataStore.java:67]: I think I changed this in a new commit. The one spacing mistake wasn't showing for me when I went to change it, but I think it is all fixed now

Comment (tsums) [reply src/main/java/codeu/model/store/persistence/PersistentDataStore.java:67]: It looks like it's still on different indentation from the rest of the neighboring code.

Are you sure you changed it?

**Review by tsums — APPROVED** [review]

**Pull request merged**